MassRegistrationOfSAPSystems/Readme.md

```diff
@@ -16,7 +16,7 @@ The example script described in this article can be run from any Powershell 5.1
     4. SAP product for the SID being registered. Valid values are S/4HANA, ECC and Other.
     5. SAP Central Services (ASCS) virtual machine resource identifier.
     6. User assigned managed identity with the [right permissions](https://learn.microsoft.com/en-us/azure/sap/center-sap-solutions/register-existing-system#setup-user-assigned-managed-identity).
-    7. Optionally, you can provide the names for Managed Resource Group and Managed Storage Account that will be created as part of the registration process. If you do not provide these names, ACSS service deploys them using its default naming convention.
+    7. Optionally, you can provide the names for Managed Resource Group,  Managed Storage Account and Managed Resources Network Access Type that will be created as part of the registration process. If you do not provide these names, ACSS service deploys them using its default naming convention.
```

Comment:
Let us reword this as below:

7. Optionally, you can provide the names for Managed Resource Group, Managed Storage Account and Managed Resources Network Access Type that will be created as part of the registration process. If you do not provide specific names, ACSS service deploys them using its default naming convention. 

- Managed Resources Network Access Type if unspecified will default to Public network access. Use 'private' as input value so that network access on managed storage account is restricted to specific virtual networks.